packages/hub/src/app/models/[owner]/[slug]/EditSquiggleSnippetModel.tsx

```diff
@@ -200,18 +206,25 @@ export const EditSquiggleSnippetModel: FC<Props> = ({ modelRef }) => {
           <VersionedSquigglePlayground
             key={playgroundKey}
             version={version}
+            sourceId={`hub:${model.owner.slug}/${model.slug}`}
```

Comment:
It would be better to move some of this elsewhere, like a config, especially for the "hub:" part.